packages/ckeditor5-fullscreen/src/handlers/abstracteditor.ts

```diff
@@ -122,4 +126,55 @@ export default class AbstractEditorHandler {
 			this._container = null;
 		}
 	}
+
+	public generatePresenceListElement(): void {
+		if ( !this._editor.plugins.has( 'PresenceListUI' ) ) {
+			return;
+		}
+
+		const presenceListWrapper = `
+			<div class="ck ck-fullscreen__left-sidebar-item">
+				<div class="ck ck-fullscreen__left-sidebar-header">Connected users</div>
+				<div class="ck ck-fullscreen__presence-list" data-ck-fullscreen="presence-list"></div>
+			</div>
+		`;
+
+		const fragment = document.createRange().createContextualFragment( presenceListWrapper );
+
+		document.querySelector( '[data-ck-fullscreen="left-sidebar-sticky"]' )!.appendChild( fragment );
```

Comment:
Maybe for consistency across this one file we'll keep the convention of creating an element and setting its `innerHTML` here and later in DO?